contrib/mathtex-script-type/mathtex-script-type.js

```diff
@@ -1,24 +1,22 @@
-/* global katex */
+import katex from "katex";
```

Comment:
Same concern as in auto-render.js.

Reply:
@kevinbarabash The module `katex` is declared as `externals` in the webpack configuration. As a result, webpack doesn't bundle the KaTeX, but tries to require the module by `require("katex")` in CJS, `define(["katex"], ...)` in AMD, and using global variable `katex`(old behavior). Basically, it add support for module loaders.

Reply:
Oh wow.  That's super cool!